config.py

```diff
@@ -1129,16 +1129,22 @@ def load_check(agentConfig, hostname, checkname):
 
 def generate_jmx_configs(agentConfig, hostname, checknames=None):
     """Similar logic to load_check_directory for JMX checks"""
-    from jmxfetch import JMX_CHECKS
+    from jmxfetch import JMX_CHECKS, get_jmx_checks
+
+    jmx_checks = JMX_CHECKS
+    auto_conf_jmx_checks = get_jmx_checks()
+    for check in auto_conf_jmx_checks:
+        if check not in JMX_CHECKS:
+            jmx_checks.append(check)
```

Comment:
This is the kind of thing we could maybe not need to do if we rely on sets, or make sure the list doesn't contain any duplicates when returning from `get_jmx_checks()`